tutorial/CodeGen/deploy/gaudi.md

```diff
@@ -1,200 +1,66 @@
-# Single node on-prem deployment with TGI on Gaudi AI Accelerator
+# Single node on-prem deployment on Gaudi AI Accelerator
 
-This deployment section covers single-node on-prem deployment of the CodeGen
-example with OPEA comps to deploy using the TGI service. We will be showcasing how
-to build an e2e CodeGen solution with the Qwen2.5-Coder-7B-Instruct,
-deployed on Intel® Gaudi® AI Accelerators. To quickly learn about OPEA in just 5 minutes 
-and set up the required hardware and software, please follow the instructions in the
-[Getting Started](../../../getting-started/README.md) section. 
+This section covers single-node on-prem deployment of the CodeGen example. It will show how to build an end-to-end CodeGen solution with the `Qwen2.5-Coder-32B-Instruct` model deployed on Intel® Gaudi® AI Accelerators. To quickly learn about OPEA and set up the required hardware and software, follow the instructions in the [Getting Started](../../../getting-started/README.md) section. 
```

Comment:
```suggestion
This section covers single-node on-prem deployment of the CodeGen example. It will show how to deploy an end-to-end CodeGen solution with the `Qwen2.5-Coder-32B-Instruct` model running on Intel® Gaudi® AI Accelerators. To quickly learn about OPEA and set up the required hardware and software, follow the instructions in the [Getting Started](../../../getting-started/README.md) section. 
```